multi_map_it_test.go

```diff
@@ -539,6 +539,49 @@ func TestMultiMap_ContainsEntry(t *testing.T) {
 	})
 }
 
+func TestMultiMap_ValueCount(t *testing.T) {
+	it.MultiMapTester(t, func(t *testing.T, m *hz.MultiMap) {
+		ctx := context.Background()
+		targetValues := []interface{}{"v1", "v2", "v3"}
+		targets := []types.Entry{
+			types.NewEntry("key1", "v1"),
+			types.NewEntry("key1", "v2"),
+			types.NewEntry("key1", "v3"),
+		}
+		for _, entry := range targets {
```

Comment:
What about using MultiMap::PutAll instead for simplicity?

Reply:
Thanks, resolved: [9ab0e17](https://github.com/hazelcast/hazelcast-go-client/pull/734/commits/9ab0e1786fcc3f4474facea567467d9e521cc78b)